flink-connector-tidb-cdc/src/main/java/org/tikv/common/codec/RowV2.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2021 TiKV Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.tikv.common.codec;
+
+import org.tikv.common.exception.InvalidCodecFormatException;
+
+import java.util.Arrays;
+
+/** Copied from client-java. */
```

Comment:
Please describe why we need copy and change this class, you can refer :

https://github.com/ververica/flink-cdc-connectors/blob/c77275cb683bdb7aa13f4d6b89baecf9e382a305/flink-connector-mysql-cdc/src/main/java/com/github/shyiko/mysql/binlog/io/BufferedSocketInputStream.java#L26

Reply:
reverted